interface/client/mistAPIBackend.js

```diff
@@ -17,15 +19,7 @@ var filterId = function(str) {
 };
 
 
-var sound = {
-    bip: document.createElement('audio'),
-    bloop: document.createElement('audio'),
-    invite: document.createElement('audio'),
-};
-sound.bip.src = 'file://'+ dirname + '/../../sounds/bip.mp3';
```

Comment:
dirname is added in the preloaded file for the mist interface. Though it might be smart to move change that do using `_dirname` as its anyway a module loaded in the preloaded file
Its a custom global variable i added. See: https://github.com/ethereum/mist/blob/develop/modules/preloader/mistUI.js#L33